2008_Czub/12C_33.2MeV/beam.dat

```diff
@@ -0,0 +1,13 @@
+*         Input file FOR023.DAT for the SHIELD Transport Code
```

Comment:
Can you remove first two lines (some old comment and empty line) ? There are meaningless for this simulation